dll/win32/shimgvw/shimgvw.c

```diff
@@ -243,20 +243,20 @@ BOOL Anime_Step(DWORD *pdwDelay)
 
 static void ZoomInOrOut(BOOL bZoomIn)
 {
-    INT i;
+    SIZE_T i;
```

Comment:
I'm not sure about using `SIZE_T` for a loop iterator variable, however let's see what anyone else can say about it.

Reply:
On the one side you remove a compiler's warning; on the other side these arrays have very few elements so that the count of these elements can hold even in a int8 or int16 😂

Reply:
`UINT` is better?